{{cookiecutter.directory_name}}/pyproject.toml

```diff
@@ -1,3 +1,21 @@
 [build-system]
 requires = ["setuptools", "wheel"]
 build-backend = "setuptools.build_meta"
+
+[tool.pytest.ini_options]
+testpaths = ["tests"]
+
+[tool.coverage.run]
+branch = true
+source = ["{{ cookiecutter.package_name }}"]
+command_line = "-m pytest"
+
+[tool.tox]
+legacy_tox_ini = """
+[tox]
+envlist = py37,py38,py39
```

Comment:
Maybe use python 3.10 instead of 3.7? I believe that e.g. the most recent numpy version no longer supports Python 3.7 (https://numpy.org/neps/nep-0029-deprecation_policy.html#drop-schedule)

Reply:
Yes, good point for another issue/PR! Adding 3.10 to the whole template (not just here in the tox config) should definitely be done. I wouldn't remove 3.7 just yet, because Python 3.7 is not EOL yet (https://endoflife.date/python) and there are also Python projects that don't use numpy (e.g. web-projects around Flask).

Reply:
Made #312.